Dockerfile

```diff
@@ -1,4 +1,14 @@
+ARG OPENVDS_IMAGE=openvds
+ARG OPENVDS_REPO=https://community.opengroup.org/osdu/platform/domain-data-mgmt-services/seismic/open-vds.git
+ARG OPENVDS_VERSION=cbcd7b6163768118805dbcd080a5b0e386b82a6a
+
+FROM alpine as utils
+ARG OPENVDS_VERSION
+RUN echo $OPENVDS_VERSION > /openvds_version.txt
```

Comment:
It's a bit fragile (and cumbersome like you pointed out) to use the openvds commit sha as cache key in GitHub action. What if something else in the Dockerfile changes? then we don't get a fresh build.

I think it would be simpler to hash the Dockerfile in gh action and use that as key. E.g: `sha1sum Dockerfile`. Then the Dockerfile itself doesn't need to "export" anything at all. radixconfig doesn't care anyway. If we also uaw the same hash as image tag and we should be all set I believe?